src/index.ts

```diff
@@ -1,40 +1,50 @@
 import { types, ExecuteWithScope, createExecuteWithScope } from "brs";
-import { glob } from "glob";
+import fastGlob from "fast-glob";
```

Comment:
just curious, did you notice any performance gain after switching to this package?

Reply:
I did not explicitly test that! buuut the reason I switched is because:
- we use `fast-glob` in `brs` instead of `glob`
- the main reason it's faster is because it uses [micromatch](https://github.com/micromatch/micromatch) rather than [minimatch](https://github.com/isaacs/minimatch), and `micromatch` has a whole section of benchmarks showing that it's faster 🤷

Reply:
cool!